packages/perseus/src/util/answer-types.ts

```diff
@@ -26,7 +26,7 @@ export type Score = {
     ungraded?: boolean;
 };
 
-/*
+/**
```

Comment:
Tiny change that surfaces all this comment goodness in VSCode and other editors when you use KhanAnswerTypes. 

## Before 

<img width="1052" alt="image" src="https://github.com/user-attachments/assets/2d4c899b-ed36-4b8b-965b-470a52f99014">


## After

<img width="1049" alt="image" src="https://github.com/user-attachments/assets/8850d0f8-fad0-4fb3-b16b-fc0ac98335c2">

Reply:
So interesting that it just needs that one asterisk! I suppose that indicates how to handle the comment, but definitely did not know this led to such a huge difference!